uelc/main/models.py

```diff
@@ -611,4 +612,27 @@ class CaseAnswerForm(forms.Form):
         return CaseAnswerForm()
 
 
+class SectionSubmission(models.Model):
```

Comment:
I'm confused by the existence of two `SectionSubmission` models. One here and one in gate_block. They seem to be basically identical, and neither have any relationships to other models in their respective apps.